target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst

```diff
@@ -0,0 +1,2 @@
+edu/ucla/library/services/utility/PropsWriter.class
```

Comment:
Did you mean to check in the `target` directory? Believe you've added it to the .gitignore up above, but perhaps forgot to `git rm` it?